CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
-# x.x.x (yyyy/mm/dd)
+# 3.0.0-rc.0 (2019/10/03)
 
-- Initial release.
+- Added SkyAgGridService and a stylesheet to create grids with default SKY UX styling and gridOptions. [#2](https://github.com/blackbaud/skyux-ag-grid/pull/2)
```

Comment:
Is there a reason we have `Added SkyAgGridService...` instead of `Added the ag grid service...`? Same question, more or less with the items that follow. I think the latter version is more consistent with how we usually reference components and services in the changelog, but if we stick with the former, I'd format them as code snippets. (Also, is `ag` correct? Or does that stand for something that we should spell out?

Reply:
I thiiiink `ag` is kind of a reference to angular? But everywhere ag grid uses it on their site it is just referenced as `ag-Grid`, ag is never spelled out or elaborated on. I wasn't sure on how to reference the service, components, etc. I can change it to be like the latter.

Reply:
Sorry for the slow response. i had a meeting and then had to run home because of school stuff. ... I updated the service and component names and made a few other doc tweaks. I also updated the date since this won't get out until Thursday. Hopefully all of the changes work, but let me know if not. Thanks!

Reply:
Oh, and I think the `ag` is short for `agnostic`.